### PR TITLE
Proposed README changes for conciseness and sentence structure.

### PR DESCRIPTION
**Pull request by DarienFlamont**
This is an assignment for my computing science technical writing class

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Getting up to speed with Python
+# Introduction to Python
```
Comment (joaoventura) [inline README.md:1]: This is the book title, could you undo the change here?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
 Pdf and epub files can be downloaded from: https://github.com/joaoventura/full-speed-python/releases/
 
-This book aims to teach the basics of the python programming language using a practical approach. Its method is quite basic though: after a very simple introduction to each topic, the reader is invited to learn by solving the proposed exercises.
+This book aims to teach the basics of the python programming language using a practical approach. The methods used in this book are basic, the reader is introduced to the topic (i.e. classes, objects, functions) then is presented with an exercise that implements the knowledge learned in the readings.
```
Comment (joaoventura) [inline README.md:5]: I don't know if I like the wording here, I present more than one exercise, and I don't particularly like the sentence "implements the knowledge learned in the readings"..

Comment (joaoventura) [reply README.md:5]: Sorry, it's quite different of my writing style, it sounds strange to me.. 😃

File: README.md (hunk 1 above)
Comment (DarienFlamont) [reply README.md:1]: Yes I will change this back.
